ReadMe.txt

```diff
@@ -1,3 +1,12 @@
+MagoMI fork:
+
+This is fork of Mago debugger which adds support of GDM MI compatible interface.
+
+See MagoMI/mago-mi/README.md for details.
+
+Original project: https://github.com/rainers/mago

```

Comment:
This note is no longer necessary in the main repo. The README might need some updates, though...